Wikidata QuickStatements.js

```diff
@@ -260,11 +259,13 @@ function zoteroItemToQuickStatements(item) {
 
 	if (item.language && (item.language.toLowerCase() in languageMapping)) {
 		let lang = item.language.toLowerCase();
+		addStatement('L' + lang, '"' + item.title + '"');
 		addStatement('P1476', lang + ':"' + item.title + '"');
 		addStatement('P407', languageMapping[lang]);
 	}
 	else {
 		// otherwise use "und" for undetermined language
+		addStatement('Len', '"' + item.title + '"');
```

Comment:
update the comment here, please: you're adding English for undetermined labels, right? Is there any reason that undetermined cant be used (I'm fairly ignorant about QS, so this may be obvious)

Reply:
I've updated.
The translator was already adding the label in English without taking into account the value in the 'language field' of a Zotero item.

I changed that, but if it still doesn't recognize the language value, it will add the label in English by default. It's better than nothing.